plugin/plugin.go

```diff
@@ -347,6 +351,19 @@ func (p *OrmPlugin) renderGormTag(field *Field) string {
 			res += fmt.Sprintf("association_jointable_foreignkey:%s;", mtm.GetAssociationJointableForeignkey())
 		}
 	}
+	if field.Type == "*gtypes.Inet" {
+		if p.dbEngine == ENGINE_POSTGRES {
+			if res == "" {
+				return "`sql:\"type=inet\"`"
+			}
+			return fmt.Sprintf("`gorm:\"%s\" sql:\"type=inet\"`", strings.TrimRight(res, ";"))
```

Comment:
`sql` tag is just the ancestor of `gorm` tag, so I think it's more preferable just to set `FieldOptions` for the field in `parseBasicFields`. Probably we want the same behavior for `uuid`.